src/react-components/room/useRoomLoadingState.js

```diff
@@ -14,18 +14,29 @@ function reducer(state, action) {
         loading: !(state.environmentLoaded && state.networkConnected),
         messageKey: state.environmentLoaded ? "enteringRoom" : "loadingObjects"
       };
+    case "environment-loading":
```

Comment:
Been a long time since I looked at this code. Is there a reason this needs its own action vs setting `enviornmentLoaded` to false in the initial reducer state (its falsey either way since its `undefined` but setting it is cleaner)?

Reply:
yeah, that's true. Removed